tests/block_stdout_child.toit

```diff
@@ -17,11 +17,11 @@ main args:
     // Repeat enough that this pipe will block until the parent reads
     // from stdout.
     1000.repeat:
-      out.write "In the beginning the Universe was created.\n"
-      out.write "This has made a lot of people very angry and been widely regarded as a bad move.\n"
+      out.out.write "In the beginning the Universe was created.\n"
```

Comment:
Maybe you should rename the locals `in` to `stdin` and `out` to `stdout`?

Reply:
done.